package.json

```diff
@@ -20,7 +20,7 @@
     "dotenv": "^16.0.3",
     "ethereum-waffle": "^4.0.2",
     "ethers": "^5.7.2",
-    "fuels": "^0.25.1",
+    "fuels": "^0.32.0",
```

Comment:
Can this not be bumped to a more current version?

Reply:
That was current a few days ago with the major pieces we needed. I went ahead and bumped to 0.33.0 though